snuba/settings/__init__.py

```diff
@@ -175,7 +175,7 @@
 SUBSCRIPTIONS_ENTITY_BUFFER_SIZE: Mapping[str, int] = {}  # (entity name, buffer size)
 
 # Temporary setting for subscription scheduler test
-SUBSCRIPTIONS_SCHEDULER_LOAD_FACTOR = 3
+SUBSCRIPTIONS_SCHEDULER_LOAD_FACTOR = 5
```

Comment:
6?

Reply:
Decided to play it safe first and go to 5, will update the description now